src/autogluon_dashboard/dashboard.py

```diff
@@ -24,26 +20,75 @@ def upload_to_s3(s3_client: botocore.client, file_name: str, object_name: str, a
         Name of object to store file contents in S3 bucket
     """
     with open(file_name, "rb") as f:
-        s3_client.upload_fileobj(f, BUCKET, object_name, ExtraArgs=args)
+        s3_client.upload_fileobj(f, bucket_name, object_name, ExtraArgs=args)
+
+
+def get_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--per_dataset_csv",
+        type=str,
+        help="Location of csv file in local filesystem to upload to S3 bucket. Example: sub_folder/file_name.csv",
+    )
+    parser.add_argument(
+        "--all_dataset_csv",
+        type=str,
+        help="Location of csv file in local filesystem to upload to S3 bucket. Example: sub_folder/file_name.csv",
+    )
+    parser.add_argument(
+        "--per_dataset_s3",
+        type=str,
+        help="Location in S3 bucket to save csv file. Example: sub_folder/file_name.csv",
+        default="dev_data/all_data.csv",
+    )
+    parser.add_argument(
+        "--all_dataset_s3",
+        type=str,
+        help="Location in S3 bucket to save csv file. Example: sub_folder/file_name.csv",
+        default="dev_data/autogluon.csv",
+    )
+
+    parser.add_argument(
+        "--s3_bucket",
+        type=str,
+        help="Name of S3 bucket that results to aggregate get outputted to",
+        default="dashboard-test-yash",
+    )
+    parser.add_argument("--s3_prefix", type=str, help="Prefix for path to results needing aggregation", default="")
+
+    args = parser.parse_args()
+    return args
 
 
 def run_dashboard():
-    per_dataset_csv_path = sys.argv[2]
-    aggregated_csv_path = sys.argv[4]
-    per_dataset_s3_loc = "dev_data/all_data.csv"
-    aggregated_s3_loc = "dev_data/autogluon.csv"
-    s3_client = boto3.client("s3")
-    global s3_url
+    args = get_args()
+    # Set variables to corrensponding command line args
+    per_dataset_csv_path = args.per_dataset_csv
+    aggregated_csv_path = args.all_dataset_csv
+    per_dataset_s3_loc = args.per_dataset_s3
+    aggregated_s3_loc = args.all_dataset_s3
+    bucket_name = args.s3_bucket
+    s3_url = f"https://{bucket_name}.s3.us-west-2.amazonaws.com/"
```

Comment:
is this s3_url used for website browsing or uploading artifacts? If it's uploading artifact, I don't think you need to provide a https url but a s3 url is enough. Raise this question becuse `us-west-2` clearly doesn't work for all buckets